camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -435,24 +436,13 @@ export default class Tasklist extends Mixins(TaskListMixin) {
   private formioUrl: string = "";
   private task: TaskPayload = {
   };
-  private setFollowup: Array<Date| null> = [];
-  private setDue: Array<Date| null> = [];
+  private setFollowup: Array<Date | null> = [];
+  private setDue: Array<Date | null> = [];
```

Comment:
an array of null doesn't seem to be a good logic? you can filter that

Reply:
When removing a date, we are setting `setFollowup` as null. That's why it's set as array of null or Date. 

Did you mean to look for a better solution, rather than current approach with your comment?

Reply:
Removed the `setFollowup and setDue` Dates.